modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -52,12 +52,33 @@ public class UpgradeCatchAllCheck {
 		_cpInstanceService.fetchByExternalReferenceCode(companyId, externalReferenceCode);
 	}
 
+	public void method(
+		String externalReferenceCode, long cProductId,
+		String cpInstanceUuid, long commercePriceListId, BigDecimal price,
+		BigDecimal promoPrice, ServiceContext serviceContext) {
+
+		_commercePriceEntryLocalService.addCommercePriceEntry(
+			externalReferenceCode, cProductId, cpInstanceUuid,
+			commercePriceListId, price, promoPrice, serviceContext);
+
+		CommercePriceEntryLocalServiceUtil.addCommercePriceEntry(
+			externalReferenceCode, cProductId, cpInstanceUuid,
+			commercePriceListId, price, promoPrice, serviceContext);
+
+		_commercePriceEntryLocalService.addCommercePriceEntry(
+			null, cpInstance.getCPDefinition().getCProductId(), cpInstance.getCPInstanceUuid(),
+			commercePriceList.getCommercePriceListId(), price, null, serviceContext);
+	 }
```

Comment:
@alberttgomes please rebase with the master and apply the new rule for the format of the tests. Sort with Sublime and run the SF on the testfile.